stairsplus/registrations.lua

```diff
@@ -158,7 +158,6 @@ minetest.register_lbm({
 			param2 = minetest.get_node(pos).param2
 
 		})
-		minetest.log('action', "LBM replaced " .. node.name ..
-			" at " .. minetest.pos_to_string(pos))
+		minetest.log('action', S("LBM replaced @1 at @2", node.name, minetest.pos_to_string(pos)))
```

Comment:
translation does not work correctly on minetest.log output
consider removing localization on this string

Reply:
What problem does it produce?, I intend to create a server and I would prefer that the logs be in Spanish

Reply:
It would be fine, but currently they appear with `@moreblocks)` in front, and it seems strings are logged in english (translation does only work in client display).